Makefile

```diff
@@ -258,7 +259,12 @@ artifacts: kustomize
 	$(KUSTOMIZE) build config/dev -o artifacts/manifests-dev.yaml
 	$(KUSTOMIZE) build config/prometheus -o artifacts/prometheus.yaml
 	@$(call clean-manifests)
-	cp -r charts artifacts/charts
+	# Update the image tag and policy
+	yq  e  '.controllerManager.manager.image.repository = "$(IMAGE_REPO)" | .controllerManager.manager.image.tag = "$(GIT_TAG)" | .controllerManager.manager.image.pullPolicy = "IfNotPresent"' -i charts/kueue/values.yaml
```

Comment:
can you add a make rule to install `yq`?

Reply:
I see it as a standard utility ... I don't think we should do it

Reply:
I don't know how much `yq` could change behavior from one version to the next.
But we can leave it as a follow up.